CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Confluent's Python client for Apache Kafka
 
+
+## v1.10.0
```

Comment:
Need help with the CHANGELOG.md version and message, the current one is sort of a placeholder.

Reply:
I'm not sure what the adopted process is here with this client. My thought is to use `## vNext`, but that's just an idea.

Reply:
Sounds reasonable to me, changed it like so.